TopMovies/DTO/GraphQL/GraphQLCategoryDTO.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  GraphQLCategoryDTO.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 01.04.2021.
+//
+
+// MARK: - GraphQLCategoryDTO
+struct GraphQLCategoryDTO: Decodable {
+  let data: DataCategoryDTO
+  struct DataCategoryDTO: Decodable {
+    let movies: MoviesCategoryDTO
+    struct MoviesCategoryDTO: Decodable {
+      let popular: EdgesCategoryDTO?
+      let nowPlaying: EdgesCategoryDTO?
+      let topRated: EdgesCategoryDTO?
+      let upcoming: EdgesCategoryDTO?
+      
+      var current: EdgesCategoryDTO? {
+        popular ?? nowPlaying ?? topRated ?? upcoming
+      }
+      
+      struct EdgesCategoryDTO: Decodable {
+        let edges: [EdgeCategoryDTO]
+        let pageInfo: PageInfoCategoryDTO
+        struct EdgeCategoryDTO: Decodable {
+          let cursor: String
+          let node: GraphQLMoviePreviewDTO
+        }
+        struct PageInfoCategoryDTO: Decodable {
+          let hasNextPage: Bool
+        }
+      }
+    }
+  }
+}
+
+extension CategoryDTOWrapper {
+  init(id: String, title: String, dto: GraphQLCategoryDTO) {
+    let dto = dto.data.movies.current
+    
+    self.id = id
+    self.title = title
+    movies = dto?.edges.map { MoviePreviewDTOWrapper.init(dto: $0.node) } ?? []
+    next = dto?.pageInfo.hasNextPage ?? true ?
```

Comment:
`dto?.pageInfo.hasNextPage ?? false`
If you're not 100% sure that you have a next page, always perform `next = nil` by default